app/Platform/AuthServiceProvider.php

```diff
@@ -5,15 +5,12 @@
 namespace App\Platform;
 
 use Illuminate\Foundation\Support\Providers\AuthServiceProvider as ServiceProvider;
+use Illuminate\Support\Facades\Auth;
 
 class AuthServiceProvider extends ServiceProvider
 {
     public function boot(): void
     {
-        // Auth::extend('api-token', function ($app, $name, array $config) {
-        //     return new TokenGuard(Auth::createUserProvider($config['provider']), $app->request, 'z', 'api_token');
-        // });
-        //
```

Comment:
I removed this commented out code. Removing the z param makes it to where even what this was doing before being commented out is pointless.

Reply:
Do we need the new `Auth` import on line 8?

Reply:
Nope, residual from me testing this comment.